tools/driver/swift_symbolgraph_extract_main.cpp

```diff
@@ -66,6 +66,10 @@ PrettyPrint("pretty-print", llvm::cl::desc("Pretty-print the resulting Symbol Gr
 static llvm::cl::opt<std::string>
 MinimumAccessLevel("minimum-access-level", llvm::cl::desc("Include symbols with this access level or more"), llvm::cl::cat(Category));
 
+static llvm::cl::list<std::string>
+ClangXCC("Xcc", llvm::cl::desc("Pass the following command-line flag to Clang"),
```

Comment:
Please don't name this `ClangXcc`, as it gets confusing since we do have some `-clang-*` flags in the driver to pass through options specifically to the clang as the linker driver.  Could you please rename this to `Xcc`?

Reply:
Sure thing.